spot_driver/spot_driver/spot_ros2.py

```diff
@@ -904,7 +956,7 @@ def step(self):
             self.node_rate.sleep()
 
 def main(args=None):
-    print('Hi from spot_driver.')
+    print(COLOR_GREEN + 'Hi from spot_driver.' + COLOR_END)
```

Comment:
This is ok to leave as a print because you don't have a node yet.

Reply:
Do you really want this line as the first line in the function, or can it be moved down a few lines to be under `create_node` so that we have the logger?

Reply:
I think this is ok to be left here so if node creation fails or something it's a little easier to track down.